src/main/java/com/google/api/codegen/grpcmetadatagen/py/PythonPackageCopier.java

```diff
@@ -72,12 +73,14 @@ public FileVisitResult postVisitDirectory(Path dir, IOException exc) {
       String outFile = inputPath.relativize(dir.resolve("__init__.py")).toString();
       // Version directory gets an empty __init__.py
       if (dir.getFileName().toString().equals(apiVersion)) {
-        docBuilder.put(outFile, Doc.text("\n"));
+        docBuilder.put(outFile, GeneratedResult.create(Doc.text("\n"), false));
 
         // All others get become namespace packages
       } else {
         docBuilder.put(
-            outFile, Doc.text("__import__('pkg_resources').declare_namespace(__name__)\n"));
+            outFile,
+            GeneratedResult.create(
+                Doc.text("__import__('pkg_resources').declare_namespace(__name__)\n"), false));
```

Comment:
In this context it's hard to figure out what the boolean parameter means. Could we either create an enum or use some other way to make it clear here?

Reply:
I can't think of any reasonable enum name for this (and would like to avoid some enum, which results into. At long last the parameter has a name, so it should be enough to describe what it does. If param name `executable` is not descriptive, I'm happy to change it to something else.

I added documentation for the `GeneratedResult<T>` and its methods to makes things less confusing.